java/pom.xml

```diff
@@ -273,6 +288,35 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>run-error-prone</id>
+      <activation>
+        <jdk>[11,)</jdk>
```

Comment:
Do we want to support jdk 9?
Or does error prone support start at 11?

Reply:
Error prone starts at 11, so luckily that means we are getting the 11 & 17 tests and I think that should be enough.